docs/msbuild/common-msbuild-project-properties.md

```diff
@@ -99,6 +100,7 @@ The following table lists frequently used properties that are defined in the Vis
 |OverwriteReadOnlyFiles|A boolean value that indicates whether you want to enable the build to overwrite read-only files or trigger an error.|  
 |PdbFile|The file name of the .pdb file that you are emitting. This property is equivalent to the `/pdb` switch of the csc.exe compiler.|  
 |Platform|The operating system you are building for. Valid values are "Any CPU", "x86", and "x64".|  
+|ProduceReferenceAssembly|A boolean value that when set to `true` enables production of [reference assemblies](https://github.com/dotnet/roslyn/blob/master/docs/features/refout.md) for the current assembly. `Deterministic` should be `true` when using this feature. This property corresponds to the `/refout` switch of the `vbc.exe` and `csc.exe` compilers.|
```

Comment:
Should we also document the CompileUsingReferenceAssemblies flag?

> If you encounter a problem using reference assemblies, you can set the boolean property CompileUsingReferenceAssemblies to false to avoid using ref assemblies even if the projects you reference produce them. This is unset by default and only ever checked against false. It is only there to provide an emergency escape hatch; a customer who hits a bug can set it to false and avoid the new codepaths.

https://github.com/dotnet/roslyn/blob/master/docs/features/refout.md#msbuild